core/vm/ovm_state_dump.go

```diff
@@ -37,6 +37,32 @@ var ovmStateDumpJSON = []byte(`
                     "stateMutability": "nonpayable",
                     "type": "constructor"
                 },
+                {
+                    "anonymous": false,
+                    "inputs": [
+                        {
+                            "indexed": false,
+                            "internalType": "bytes32",
+                            "name": "msgHash",
+                            "type": "bytes32"
+                        }
+                    ],
+                    "name": "RelayedMessage",
+                    "type": "event"
+                },
+                {
+                    "anonymous": false,
+                    "inputs": [
+                        {
+                            "indexed": false,
+                            "internalType": "bytes",
+                            "name": "message",
+                            "type": "bytes"
+                        }
+                    ],
+                    "name": "SentMessage",
+                    "type": "event"
+                },
```

Comment:
Are these intentionally added? Or was this when we were experimenting last night?

Reply:
Omg great didn't see the event! LGTM!